indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -577,30 +577,60 @@ th.i-table {
     }
 }
 
-.recipients {
-    @extend .flexrow;
-    justify-content: space-between;
+#form-group-recipients {
+    position: relative;
 
-    .static-text-widget {
-        width: inherit;
+    .recipients-box {
+        @include border-all($pastel-gray);
+
+        display: inline-block;
+        padding-left: 5px;
+        border-radius: 2px;
```

Comment:
Use `@include border-radius()`
Check the defaults in `indico/htdocs/sass/base/_defaults.scss`, the default radius is already `2px`.